### ISSUE TITLE
Use official wunderlist node-module

### ISSUE THREAD

**Issue by Thor77** [wayneashleyberry/wunderlist-cli#99]
It seems like wunderlist provides an official supported [node-module](https://www.npmjs.com/package/wunderlist) (since 03/2015) for easy api-access.
We should check if it provides all the features we need and maybe rewrite parts of Wunderline on top of it.

**Comment by wayneashleyberry** [comment]
Yeah, I seem to remember not using it for some reason... but it's definitely worth re-evaluating.
